net-analyzer/iplog/files/iplog-2.2.3-C23.patch

```diff
@@ -22,15 +21,22 @@ https://bugs.gentoo.org/712644 - type aliaces for musl
  
  #ifdef HAVE_PATHS_H
  #	include <paths.h>
-C23 and GNU-15 compatibility, explicitly cast sockaddr
---- a/src/iplog_tcp.c
-+++ b/src/iplog_tcp.c
-@@ -144,7 +144,7 @@
+Explicitly cast sockaddr_in to sockaddr, always correct thing to do in this situation.
+--- a/src/iplog_tcp.c	2025-02-16 22:16:15.486203469 +0400
++++ b/src/iplog_tcp.c	2025-02-16 22:16:58.045309444 +0400
+@@ -142,14 +142,11 @@
+ 		fn_sin.sin_port = tcp->th_sport;
+ 		fn_sin.sin_addr.s_addr = ip->ip_src.s_addr;
  
  		ret = sendto(raw_sock, (char *) xip,
  				sizeof(struct ip) + sizeof(struct tcphdr), 0,
 -#if !defined(__GLIBC__) || (__GLIBC__ < 2)
-+#if !defined(__GLIBC__) || (__GLIBC__ < 2) || (__STDC_VERSION__ > 201710L)
```

Comment:
Also, sorry, I meant to catch this before and I even remembering planning to comment on it, and apparently I didn't. :|